eZ/Publish/API/Repository/Tests/UserServiceTest.php

```diff
@@ -1084,6 +1079,118 @@ public function testCreateUserThrowsInvalidArgumentException()
         $this->fail('Expected ValidationError messages did not occur.');
     }
 
+    /**
+     * Test for the createUser() method.
+     *
+     * @covers \eZ\Publish\API\Repository\UserService::createUser
+     * @depends eZ\Publish\API\Repository\Tests\UserServiceTest::testCreateUser
+     */
+    public function testCreateUserWithEmailAlreadyTaken()
+    {
+        $repository = $this->getRepository();
+
+        $userContentType = $this->createUserContentTypeWithAccountSettings('user_email_unique', [
+            Type::REQUIRE_UNIQUE_EMAIL => true,
+        ]);
+
+        $existingUser = $this->createUserVersion1(
+            'existing_user',
+            'unique@email.com',
+            $userContentType,
+        );
+
+        $editorsGroupId = $this->generateId('group', 13);
+        /* BEGIN: Use Case */
+        // $editorsGroupId is the ID of the "Editors" user group in an eZ
+        // Publish demo installation
+
+        $userService = $repository->getUserService();
+
+        // Instantiate a create struct with mandatory properties
+        $userCreate = $userService->newUserCreateStruct(
+        // admin is an existing login
```

Comment:
Nitpick: Wrong indent

```suggestion
            // admin is an existing login
```